src/Interfaces/AbstractPolyhedron_functions.jl

```diff
@@ -253,8 +253,7 @@ function remove_redundant_constraints(constraints::AbstractVector{S};
     if remove_redundant_constraints!(constraints_copy; backend=backend)
         return constraints_copy
     else  # the constraints are infeasible
-        N = eltype(first(constraints))
-        return EmptySet{N}(dim(constraints[1]))
+        return nothing
```

Comment:
maybe lets instead return an array of the same type as `constraints` if it failed to find a reduction.

Reply:
done